examples/AdvancedOperations/AddSmartDisplayAd.cs

```diff
@@ -147,6 +149,7 @@ private static string CreateBudget(GoogleAdsClient client, long customerId)
         /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
         /// <param name="budgetResourceName">The campaign budget resource name.</param>
         /// <returns>Resource name of the newly created campaign.</returns>
+        // [START AddSmartDisplayAd_1]
```

Comment:
Is there a better/more specific name that we can apply for these examples with multiple region tags? Perhaps including the method name, something like `AddSmartDisplayAd_Run`, `AddSmartDisplayAd_CreateBudget`, etc? I think it'd be a lot easier to maintain the documentation links in the future.

Reply:
I agree and I considered this when writing the script that generated these changes but it was too much effort compared to the benefit. I am happy to discuss this further offline if needed.